src/main.cpp

```diff
@@ -1640,7 +1644,11 @@ void vcRenderSceneUI(vcState *pProgramState, const ImVec2 &windowPos, const ImVe
 
       // TODO: Putting this here for now
       if (pProgramState->settings.activeViewportCount > 1)
+      {
+        ImGui::Checkbox(udTempStr("%s##firstViewportMapMode", vcString::Get("orthographicCameraViewport")), &pProgramState->settings.camera.mapMode[0]);
+        ImGui::SameLine();
         ImGui::Checkbox(udTempStr("%s##secondViewportMapMode", vcString::Get("orthographicCameraViewport")), &pProgramState->settings.camera.mapMode[1]);
```

Comment:
This is ugly UX; I don't have an alternative at the moment but this will be confusing

Reply:
I have added 1 & 2 at the end of the checkbox label.
This is just temporary, currently working on adding visualisations effect per viewport. Once it is done, will move the map mode to the tab of each viewport in the visualisation menu.